### ISSUE TITLE
Theme for popup window?

### ISSUE THREAD

**Issue by Paracells** [ADMARIl/halcyon-jetbrains#9]
Popup window have default theme from Darcula

![color](https://user-images.githubusercontent.com/575811/116055795-be63ba00-a685-11eb-872b-7ee682b3befa.png)

i found how to fix it and added 
`Settings -> Color Scheme -> General -> Popups and Hints -> Documentation` - **Background: #2F3B54 (from original theme)**

Result
![line](https://user-images.githubusercontent.com/575811/116056103-126e9e80-a686-11eb-9a86-df68450dac9c.png)

but problem with line (marked), it should be more darker or lighter? I don't find how to fix it.

**Comment by ADMARIl** [comment]
I agree that this change looks a lot better. Unfortunately as you might’ve already experienced there’s not the best documentation out there on how to set these kind of values. If I get some time this weekend I’ll spend some time digging to see if this is possible

**Comment by Paracells** [comment]
In theme.json file need to change

```
"CompletionPopup": {
      "selectionInactiveBackground": "#1d2433", (now #3a3a3a)
      "selectionBackground": "#1d2433",      (need add)
    },
```
color taken from original theme

![Untitled](https://user-images.githubusercontent.com/575811/116502872-5b5c6800-a8bd-11eb-82f8-a4f3275f69d5.png)

**Comment by ADMARIl** [comment]
I've implemented this change in [2.1.0](https://github.com/ADMARIl/halcyon-jetbrains/releases/tag/v2.1.0) which should be available for download from JetBrains within a few days. If you want the update before then I've included the plugin zip in the linked release. Thanks for your help!
